src/lib/protocols/ssh.c

```diff
@@ -54,23 +59,46 @@
   that usually is packet 14
 */
 
-/* #define SSH_DEBUG 1 */
+#define SSH_DEBUG 1
```

Comment:
Please disable that `#define`.
My nDPI integration does not like writing stuff to `stdout`.